src/pytest_codspeed/plugin.py

```diff
@@ -241,11 +246,11 @@ def __call__(self, func: Callable[..., Any], *args: Any, **kwargs: Any) -> Any:
         plugin.benchmark_count += 1
         if plugin.is_codspeed_enabled and plugin.should_measure:
             assert plugin.lib is not None
-            _run_with_instrumentation(
+            return _run_with_instrumentation(
                 plugin.lib, self._request.node.nodeid, func, *args, **kwargs
             )
```

Comment:
You can run the tests with the env variable  `CODSPEED_ENV` set to 1. We run this path when the code is instrumented with valgrind but since we don't have the local CLI (yet :smile: ) and doing it directly with valgrind is painful, it's not super easy to do locally for now. It's checked in the CI though.

Reply:
ah, I'm not on linux so I can't test it 😊